api/environments/models.py

```diff
@@ -10,7 +10,7 @@
 from django.db import models
 from django.db.models import Prefetch, Q
 from django.utils import timezone
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
```

Comment:
I had a look at where we're actually using this and I think we can just remove it, right?

Reply:
Yeah, we don't really support any other language; I will remove them

Reply:
Done